pontoon/base/static/js/translate.js

```diff
@@ -2666,7 +2686,6 @@ var Pontoon = (function (my) {
           entity: entity.pk,
           translation: serializedTranslation,
           plural_form: submittedPluralForm,
-          original: entity['original' + self.isPluralized()],
```

Comment:
The original string can be `Entity.string` or `Entity.string_plural`, depending on which [plural form](https://pontoon.mozilla.org/sl/amo-frontend/all-resources/?search=count&string=168317) you translate (note how the source string changes as you switch tabs). Which means https://github.com/mozilla/pontoon/pull/856/files#diff-4e362b41ba93a00a7c17060896d1a8caR38 is incorrect for pluralized strings, because it always takes `Entity.string` as the source string.

You can either keep the parameter or implement the logic in python that will tell you which source string to use based on `Translation.plural_form`.

Reply:
`compare-locales` doesn't currently support `.po` files, so I'll resture that one to keep Translate Toolkit checks intact.